xmlStyle_test.go

```diff
@@ -60,6 +60,8 @@ func (x *XMLStyleSuite) TestMarshalXlsxStyleSheetWithAFill(c *C) {
 }
 
 // Test we produce valid output for a style file with one border definition.
+// Test has been modified, because of Excel quirks, you need to provide empty
+// elements.
```

Comment:
I don't think it's useful to say what this function used to do. It should always only be a comment about what this function does. e.g.

```golang
// Test we produce valid output for a style file with one border definition.
// Empty elements are required to accommodate for Excel quirks.
```

^ I'm not exactly sure what it means, but along those lines, you as the author probably know better what to write :).